### DISCUSSION TITLE
libgl configuration issues on install

### DISCUSSION THREAD

**Discussion by guido-granda** [pyvista/pyvista#4313] [Q&A] [answered]
Hello, 
I'm trying to run the examples of pyvista with trame located here: [https://github.com/pyvista/pyvista/tree/main/examples_trame](https://github.com/pyvista/pyvista/tree/main/examples_trame)
My computer meets the requirements specified in the file requirements.txt and I'm able to run the example PyVistaLocalView.py, however, when I try running all the others I get this message in the terminal: 

App running at:
 - Local:   http://localhost:8080/
 - Network: http://127.0.1.1:8080/

Note that for multi-users you need to use and configure a launcher.
And to prevent your browser from opening, add '--server' to your command line.
libGL error: MESA-LOADER: failed to open swrast: /usr/lib/dri/swrast_dri.so: cannot open shared object file: No such file or directory (search paths /usr/lib/x86_64-linux-gnu/dri:\$${ORIGIN}/dri:/usr/lib/dri, suffix _dri)
libGL error: failed to load driver: swrast
X Error of failed request:  BadValue (integer parameter out of range for operation)
  Major opcode of failed request:  151 (GLX)
  Minor opcode of failed request:  3 (X_GLXCreateContext)
  Value in failed request:  0x0
  Serial number of failed request:  74
  Current serial number in output stream:  75

And the display window open and says connection closed.

I'm using 
pyvista   0.38.3
 vtk    9.2.6 
matplotlib    3.7.1 
trame    2.3.2  
trame-client    2.7.5                   
trame-components          2.1.0         
trame-deckgl              2.0.1                
trame-markdown            2.0.2          
trame-matplotlib          2.0.1             
trame-plotly              2.1.0                 
trame-rca                 0.3.0                   
trame-router              2.0.1                
trame-server              2.10.0              
trame-simput              2.3.1               
trame-vega                2.0.2                 
trame-vtk                 2.4.4                   
trame-vuetify             2.2.4    
Do you know how to solve this? 
Thanks

**Comment by banesullivan** [comment, marked as answer]
This looks like an issue with libgl on your system. This is likely independent of VTk/PyVista. Maybe this thread helps: https://askubuntu.com/questions/1352158/libgl-error-failed-to-load-drivers-iris-and-swrast-in-ubuntu-20-04

**Reply by guido-granda** [reply]
Yes it was a issue with my system. Thank you.
